docs/architecture/assets.md

```diff
@@ -1,58 +1,81 @@
 
 ![Architecture core concepts](../img/architecture/asset/asset-issuance.png)
 
-### Fungible assets
+### Type
+
+#### Fungible asset
 
-A fungible asset is encoded using the amount and the `faucet_id` of the faucet which issued the asset. The amount is guaranteed to be  `$2^{63} - 1$ or smaller, the maximum supply for any fungible asset. Examples of fungible assets are ETH and stablecoins, e.g., DAI, USDT, and USDC.
+Fungible assets are encoded with the amount and the `faucet_id` of the issuing faucet. The amount is always `$2^{63} - 1$` or smaller, representing the maximum supply for any fungible asset. Examples include ETH and various stablecoins (e.g., DAI, USDT, USDC).
 
-If the `faucet_id` of MATIC is `2`, 100 MATIC are encoded as `[100, 0, 0, 2]`; the `0`s in the middle distinguish between fungible and non-fungible assets.
+If the `faucet_id` of ETH is `2`, 100 ETH is encoded as:  
+`[100, 0, 0, 2]`
 
-### Non-fungible assets
+The zeros in the middle positions distinguish fungible from non-fungible assets.
 
-A non-fungible asset is encoded by hashing the asset data into a `word` and then replacing the second element with the `faucet_id` of the issuing account: For example `[e0, faucet_id, e2, e3]`. Note that the second element is guaranteed to be non-zero. Together with the fungible asset encoding, this makes it easy to differentiate between both asset types by inspecting the second element.
+#### Non-fungible asset
 
-Examples of non-fungible assets are all NFTs, e.g., a DevCon ticket. The ticket's data might be represented in a JSON string representing which DevCon, the date, the initial price, etc. Now, users can create a faucet for non-fungible DevCon tickets. This DevCon faucet would hash the JSON string into a `word` to transform the ticket into an asset.
+Non-fungible assets are encoded by hashing the asset data into a `Word` and placing the `faucet_id` as the second element. Examples include NFTs like a DevCon ticket.
+
+A non-fungible asset is encoded as:  
+`[e0, faucet_id, e2, e3]`
+
+The `faucet_id` at position `1` distinguishes non-fungible from fungible assets.
 
 ### Storage
 
-[Accounts](accounts.md) and [notes](notes.md) contain asset vaults that are used to store assets. Accounts can keep unlimited assets in a sparse Merkle tree called `account vault`. Notes can store up to `255` distinct assets.
+[Accounts](accounts.md) and [notes](notes.md) have vaults used to store assets.
 
 ![Architecture core concepts](../img/architecture/asset/asset-storage.png)
 
-The information on which and how many assets are owned can be private depending on the account's or note's storage mode. This is true for any native asset in Miden.
+### Burning
+
+Assets in Miden can be burned through various methods, such as rendering them unspendable by storing them in an unconsumable note, or sending them back to their original faucet for burning using it's dedicated function.
```

Comment:
Should `Asset`s be written in monospace, similar to how `Account` and `Note` are written in monospace on their respective pages?